x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -599,11 +601,10 @@ export const MLInferenceLogic = kea<
               [];
             const mlModel = supportedMLModels.find((model) => model.model_id === modelId);
             const modelType = mlModel ? getMLType(getMlModelTypesForModelConfig(mlModel)) : '';
-            const disabledReason = getDisabledReason(
-              missingSourceFields,
-              indexProcessorNames,
-              pipelineName
-            );
+            const disabledReason =
```

Comment:
Since the only remaining reason for disabling a selection option is if some fields are missing, I deleted the function from `utils.ts` and moved the evaluation here.